tests/groups.rs

```diff
@@ -41,13 +41,9 @@ fn required_group_missing_arg() {
     assert_eq!(err.kind, ErrorKind::MissingRequiredArgument);
 }
 
-// This tests a programmer error and will only succeed with debug_assertions
-// #[cfg(debug_assertions)]
+#[cfg(debug_assertions)]
```

Comment:
Another `#[cfg(debug_assertions)]` (and a few below)

Reply:
They are all needed because otherwise release profile tests would fail.